requirements/develop.in

```diff
@@ -1,9 +0,0 @@
--r test.in
--c test.pip
-
-invoke==1.3.0
-wheel
-# Pinned Twine's version. The newest one pulls a version of the cryptography pkg that's make the CircleCi build fail.
-twine==3.3.0
-pre-commit==2.14.0
-pip-tools==6.2.0
```

Comment:
We could remove the pre-commit conf (and documentation).
I don't think we would a need one, since `poetry.lock` is updated when adding or updating deps?